### DISCUSSION TITLE
CNC UGS locking up

### DISCUSSION THREAD

**Discussion by KAR4030** [winder/Universal-G-Code-Sender#2199] [General]
I am hoping someone can help.  I have a Genmitsu 4030 CNC machine.  I have accomplished several projects on the machine using VCarve & UGS.  Yesterday while doing another project the machine just stopped moving while spindle spinning mid-project.  UGS locks up and the only way to communicate is to go to Task Manager and End Task.  Then restart UGS and then home (this retracts the bits and goes to home) This has happened over 30 times in the last two days.  It stops at different points in the process.   Sometimes 3 seconds into the program.  Sometimes 2 minutes, sometimes 5 minutes.
 Here is my configuration.
Machine+ Genmitsu 4030
Computer Dell (6 months old) Windows 11
UGS 2.0.12  for 64 bit

I have searched over and tried everything I can find.  So far I have tried
1.  Reloaded VCarve program
2. Re-accomplished tool paths and loaded that new program.
3. Shut down computer and restart.
4. Shut down machine processor and start new
5. Adjusted feeds/speeds
6. Shielded USB cable with Toroid
7. Moved computer away from machine
8. Cleared Cache
9. Ensured Comm port was not powering down
10. Everything in shop is turned off (noise)
11. Reloaded the latest UGS64 2.0.14
12. Loaded UGS 2.0.14

I am at my wits end after spending two days troubleshooting this.  All with No Fix.  I am about to dump UGS and move to another GCode Sender, but Thought I would ask first.

Best Regards,

KAR4030

**Comment by chrisla23** [comment]
Is it particularly complicated / 3D carve? If so, adjusting the RAM given to the UGS process could be something to try.

-Chris

**Comment by KAR4030** [comment]
Chris,

Nothing complicated.  It is a profile outer edge cut.  It also does it on a drill program drilling two holes.

Thanks,


Kim
________________________________

**Comment by KAR4030** [comment]
I found the problem - bad USB cable.

Kim Rutherford

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
